### PR TITLE
incoming: add configmap to store incoming bug reports

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/operator/reporters/incoming/incoming_api.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+	Timestamp time.Time `json:"timestamp"`
+
+	// Count daily incoming numbers for components
+	Components []ComponentCount `json:"components"`
+
+	// Count daily incoming numbers for severity
+	Severities []SeverityCount `json:"severities"`
+}
+
+type IncomingReport struct {
```
Comment (sttts) [inline pkg/operator/reporters/incoming/incoming_api.go:39]: what is it doing? godoc

File: pkg/operator/reporters/incoming/incoming_api.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (sttts) [inline pkg/operator/reporters/incoming/incoming_api.go:11]: order

File: pkg/operator/reporters/incoming/incoming_api.go (hunk 1 above)
Comment (sttts) [reply pkg/operator/reporters/incoming/incoming_api.go:39]: oh, that's the api

Comment (mfojtik) [reply pkg/operator/reporters/incoming/incoming_api.go:39]: yeah, the file is named _api :-)

Comment (sttts) [reply pkg/operator/reporters/incoming/incoming_api.go:39]: I am used to types.go ;-)

**Pull request merged**